transformations/dev-prod-mode/dev-prod-mode.md

```diff
@@ -0,0 +1,285 @@
+
+{: .image-popup}
+![Screenshot - Configuration Changes Differences](/transformations/dev-prod-mode/24-config-changes-diff.png)
+
+The conflict here means that the production project changed while we were working in the development branch. 
+`Hello world!` was changed to `Hello tester!`. If you merged the development project to production now, 
+the development project would overwrite any changes made in production. In our case, `Hello tester!` would become
+`Hello world!` again.
+
+### Resolve Conflicts
+To resolve the conflict and merge the development branch to production safely without overwriting anything new, 
+you need to manually replicate the production change in the development branch and turn `Hello world!` to `Hello 
+tester!` too.
+
+{: .image-popup}
+![Screenshot - Match Change from Production](/transformations/dev-prod-mode/25-match-change-in-prod.png)
+
+Don’t forget to click the blue check icon on the right. When done, go back to the dashboard and look 
+at the project diff again. 
+
+{: .image-popup}
+![Screenshot - Remaining Changes Diff](/transformations/dev-prod-mode/26-config-remaining-changes-diff.png)
+
+You will still see the configuration as in conflict because it was edited both in production as well 
+as in the development branch. However, we had the option to edit the changes and make sure the remaining 
+differences are only those that we really want. In this case, we only want the two following changes.
+
+{: .image-popup}
+![Screenshot - Config Changes - Variable](/transformations/dev-prod-mode/18-diff-config-changes-var.png)
+
+and
+
+{: .image-popup}
+![Screenshot - Config Changes - Shared Code](/transformations/dev-prod-mode/17-diff-config-changes-shared-code.png)
+
+## Merge to Production
+Now click **MERGE TO PRODUCTION** and confirm. If the merge is successful, you will be taken to your production 
+project. The development branch is deleted when merged. 
+
+Go to your transformation in your production project and let’s see what did or did not change:
+
+In `outFile`, there is still the `Hello tester!` as we changed it in production earlier; it was not overwritten by 
+`Hello world!` that was originally in the development branch. The file `demoBranchFile.txt` was successfully 
+copied from the development branch.
+
+{: .image-popup}
+![Screenshot - Check Production Changes](/transformations/dev-prod-mode/27-check-changes-in-prod-var.png)
+
+The new part of the shared code that you added in the development branch was copied from the development branch as 
+well.
+
+{: .image-popup}
+![Screenshot - Check Production Changes](/transformations/dev-prod-mode/28-check-changes-in-prod-code.png)
+
+*Note: All events in the development branch are separated from the events in production. They are only displayed 
+in the branch where they took place.* 
+
+
+Now let’s do the same thing with tables to see how table storage behaves in Development/Production Mode. 
+
+*To be continued.*
```

Comment:
to ukazování na files je strašně nešťastný, to je funkce, kterou požívá asi tak 1% uživatelů - to se musí předělat na tabulky

Reply:
jj, to Tomas F. taky rikal (kdyz to po mne prochazel) s tim, ze se to bude muset dodelat (demo prej bylo jenom pro sporku)

Chtela jsem Kristyne s TN vyhovet, odlozila jiny veci a do noci se s tim pres dva dny patlala. Zbytecne :-)).

Tohle se mi nikdy jako task dostat do ruky nemelo. Nevedela jsem, do ceho jdu. Tomas Netrval mi pres Kristynu zadal neco, co bylo evidentne dost mimo (mel vedet, ze to demo je o files a ne o tabulkach), a to zrejme jenom proto, ze bylo jasny, ze nikdo jinej tutorial k tem dvema features mimo priority takhle rychle neudela (hodne chvatali, protoze to potrebovali pro linkovani z nejakych in-app announcements). Jeste chteli, abych delala screenshoty primo z videa. Krome toho, ze by byly rozmazany a necitelny, pouzivalo se tam pojmenovani typu CSAS_demo, apod.  

U tutorialu pro MLflow model depl. jsem dosla k zaveru, ze vubec nemam sanci to podle poslanyho videa jakkoliv rozumne udelat, a Jakub M. rekl, ze by se k tomu dostal nejdriv pristi stredu. Takze to nakonec zacal delat Tomas N. ve ctvrtek rano sam.
Sorry, tohle sem asi moc nepatri. A vim, ze stekam na spatnej strom :-). Jen jsem to potrebovala nekomu rict, protoze se mi to vubec nelibi. 

Tak co dal? Mohl by to nekdo dodelat po mne? Nebo to mam predelat na tabulky ja? Jen mi tu lezi par veci, co bych potrebovala udelat pred tim.

Reply:
Mergni to, pak to nějak předěláme
Navíc potřebujem dořešit tohle https://keboola.slack.com/archives/C01DDD1QMN3/p1613771059019100